src/Standards/Generic/Tests/Functions/OpeningFunctionBraceKernighanRitchieUnitTest.1.inc

```diff
@@ -212,3 +212,14 @@ function myFunction($a, $lot, $of, $params)
 function myFunction() {}
 function myFunction()      {} // Too many spaces with an empty function.
 function myFunction()	{} // Too many spaces (tab) with an empty function.
+
+// phpcs:set Generic.Functions.OpeningFunctionBraceKernighanRitchie checkFunctions 0
+function shouldBeIgnored()
+{}
+// phpcs:set Generic.Functions.OpeningFunctionBraceKernighanRitchie checkFunctions 1
+
+function dnfReturnType(): (Response&SuccessResponse)|AnotherResponse|string
+{}
+
+function contentAfterOpeningBrace() { // Some comment.
+}
```

Comment:
What about adding a second variation of this test with non-comment content after the brace ?

Reply:
:+1: